src/components/AddNewBook.js

```diff
@@ -1,18 +1,56 @@
-import React from 'react';
+/* eslint-disable consistent-return */
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { useDispatch } from 'react-redux';
+// import your Action Creators
```

Comment:
- [ ] Please remove this commented code.

Reply:
Done